docs/src/templates/pages/documentation/css/structure.hbs

```diff
@@ -6,112 +6,134 @@ docs: true
 
-This minimal approach has two goals; to provide clear structure and to define CSS conventions – such as a consistent naming scheme – to which the development team should adhere to.
+The main goals of this structure are to provide clear structure and to define CSS conventions to which the development team should adhere to (such as a consistent naming scheme).
 
 Our Sass files are structured so that it is clear which styles are defined as base styles – intended to be extended upon – and which styles are the components or modules that extend this base styling.
 
-The `scss` folder contains the following directories:
+
+<a name="folder-structure"></a><h2 class="sg-sectionHeading">SCSS Folder structure</h2>
+
+Our SCSS structure builds on ideas discussed in methodologies such as [ITCSS](https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/) and [SMACSS](https://smacss.com/).
+
+The `scss` folder in each module is structured as follows:
 
 ```
 scss
 ┣ base
 ┃ ┗ [...]
-┣ functions
+┣ components
+┃ ┗ [...]
+┣ objects
 ┃ ┗ [...]
-┣ mixins
+┣ settings
 ┃ ┗ [...]
-┣ partials
-┃ ┣ [...]
-┃ ┗ components
-┃   ┗ [...]
-┃ ┗ modules
-┃   ┗ [...]
-┣ views
+┣ tools
+┃ ┗ [...]
+┣ trumps
 ┃ ┗ [...]
 ```
 
-<hr>
+To see the structure of the SCSS directory in the base fozzie module, [check it out on github](https://github.com/justeat/fozzie/tree/master/src/scss).
 
-### /base
 
-The [base](https://github.je-labs.com/ICP/app_consumerweb/tree/master/src/ConsumerWeb/Assets/src/scss/base) folder is where the styles that make up the foundation of the project are defined.  For example, [reset](https://github.je-labs.com/ICP/app_consumerweb/tree/master/src/ConsumerWeb/Assets/src/scss/base/_reset.scss) and [typographic](https://github.je-labs.com/ICP/app_consumerweb/tree/master/src/ConsumerWeb/Assets/src/scss/base/_typography.scss) styles are defined here, as are files containing globally relevant [CSS variables](](https://github.je-labs.com/ICP/app_consumerweb/tree/master/src/ConsumerWeb/Assets/src/scss/base/_variables.scss)) and [colors](https://github.je-labs.com/ICP/app_consumerweb/tree/master/src/ConsumerWeb/Assets/src/scss/base/_color-palette.scss).
+<h3 class="sg-sectionHeading">Base</h3>
 
-<hr>
+The [base folder](https://github.com/justeat/fozzie/tree/master/src/scss/base) is where the styles that make up the foundation of the module are defined.
 
-### /partials
+For example, typographic styles can be defined here, as can base layout, reset and print styles.
 
-The [partials](https://github.je-labs.com/ICP/app_consumerweb/tree/master/src/ConsumerWeb/Assets/src/scss/partials) folder contains two folders; `/components` and `/modules`.
+n.b. We use [Normalise.css](https://github.com/justeat/fozzie/blob/master/src/scss/_dependencies.scss#L36) over a standard reset stylesheet, but styles that are need on top of this to 'reset' element styling can be added in a module's reset file here.
```

Comment:
Use the canonical URL here in case the code changes to ensure the user is taken to the correct line.